docs/userGuide/gitignoreFile.md

```diff
@@ -45,3 +45,11 @@ _site/
 .idea/*
 *.iml
 ```
+
+<box type="info">
+
+All pre-existing `.gitignore` file will not be overwritten if you are converting using the `markbind init --convert` command. 
```

Comment:
```suggestion
A pre-existing `.gitignore` file will not be overwritten if you are using the `markbind init --convert` command. 
```

Reply:
Updated in [6f09122](https://github.com/MarkBind/markbind/pull/2248/commits/6f091229ca498782376f4be82f5e6b3b1257c035)!